INSTALL

```diff
@@ -0,0 +1,121 @@
+--------------
+There are unit tests available for the persistency client library component available.
+The unit tests are used to verify that the component is working correctly and exclude any
+regressions. Please refer always to the source codeto see the available tests
+
+Preconditions:
+--------------
+* GENIVI Lifecycle components are available on thesystem
+  * The GENIVI lifecycle components are available here: 
+     http://projects.genivi.org/node-state-manager/
+     http://projects.genivi.org/node-startup-controller/
+  * Make sure Node State Manager is running (Start "node-state-manager")
+* Persistence Administration Service is available in the system and running
+  * The GENIVI Persistence Administration Service is available here:
+   http://git.projects.genivi.org/?p=persistence/persistence-administrator.git
+* Persistence Common Object is available in the system
+  * The GENIVI Persistence Common Object is available here:
+   http://git.projects.genivi.org/?p=persistence/persistence-common-object.git
+   use the key-value-store backend (run configure step with "--with_database=key-value-store")
+* Persistence partitions has been created and mounted and test data is
+* If partition and data is not available, use the PAS to setup the test data.
+  * make sure node-state-manager and pers_admin_svc are running
+  * Now install the test data:
+   "persadmin_tool install /path_to_test_data/PAS_data.tar.gz"
+   The test data is provided by the PCL (../test/data/PAS_data.tar.gz)
+* Make sure dlt-daemon is running
+* Make sure D-Bus system bus is available 
+
+Application verification
+------------------------
+To check if an application is valid/trusted and allowed to access persistent data an application check 
+functionality can be enabled (configure step --enable-appcheck, see section 6 “How to build”).
+If an application is not valid/trusted every API call to the key-value or file API returns the 
+error EPERS_SHUTDOWN_NO_TRUSTED will be returned.
+A trusted/valid application has a corresponding resource configuration table installed by the 
+Persistence Administration Service under /Data/mnt-c or /Data/mnt-wt.
+
+Data location and partitions
+----------------------------
+Persistence expects its data under the two folders /Data/mnt-c and /Data/mnt-wt.
+It is requested to have one persistence partition which will be mounted to the two folders stated above (one partition will be mounted to two different mointpoints).
+
+Example:
+sudo mount -t fsType /dev/sdx /Data/mnt-wt
+sudo mount -t fsType /dev/sdx /Data/mnt-c
+
+Run tests:
+----------
+run persistency unit test "./persistence_client_library_test" 
+
+Expected test result:
+---------------------
+The expected result is to have 0 failures and 0 errors, see example output below:
+
+./persistence_client_library_test
+Running suite(s): Persistency client library
+100%: Checks: 13, Failures: 0, Errors: 0
+persistence_client_library_test.c:141:P:GetData:test_GetData:0: Passed
```

Comment:
I think this is unnecessary and too verbose.  If the intention is to give an example, it should be enough to include up to the 100% line.  
If the intention is to show exactly the expected output then this would have to change every time the test suite changes!  Unnecessary, in my opinion (but of course the component maintainer decides).